packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -13,15 +13,26 @@ import {
 } from "@src/modules/userContext";
 import { getEnvironmentCreator } from "@src/modules/environment";
 import { ServerState } from "@models/server/ServerState";
+import { getUpdateNotificationCreator } from "./update";
+import { VersionInfo } from "server/src/api/versions";
 
 const useSliceMachineActions = () => {
   const dispatch = useDispatch();
 
+  // Update Notification
+
+  const updateNotification = (payload: VersionInfo) =>
+    dispatch(getUpdateNotificationCreator(payload));
+
   // Modal store
   const closeLoginModal = () =>
     dispatch(modalCloseCreator({ modalKey: ModalKeysEnum.LOGIN }));
   const openLoginModal = () =>
     dispatch(modalOpenCreator({ modalKey: ModalKeysEnum.LOGIN }));
+  const closeUpdateModal = () =>
```

Comment:
Name of the action creator should be coherent with the modal key : `closeNewVersionModal`